README.md

```diff
@@ -29,6 +29,15 @@ If you use the [Axiom CLI](https://github.com/axiomhq/cli), run `eval $(axiom co
 
 Otherwise create a personal token in [the Axiom settings](https://cloud.axiom.co/settings/profile) and export it as `AXIOM_TOKEN`. Set `AXIOM_ORG_ID` to the organization ID from the settings page of the organization you want to access.
 
+You can also configure the client using options passed to the axiom.NewClient function:
```

Comment:
`axiom.NewClient` is Axiom Go, specific, no? Can you find the appropriate wording for NodeJS? Like `[...] passed to the constructor for the Client:`.

Reply:
good catch, fixed it.